### PR TITLE
feat(multiple-provider-form): create component for step2

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA
desktop view
image

mobile view
image

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/collector-create/modules/MultipleProviderForm.vue [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+@screen tablet {
+    .multiple-provider-form {
+        width: 100%;
+        .radio-container {
+            display: none;
+        }
+        .dropdown-container {
+            @apply flex gap-4;
+        }
+    }
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/collector-create/modules/MultipleProviderForm.vue:110]: praise: !!

File: apps/web/src/services/asset-inventory/collector/collector-create/modules/MultipleProviderForm.vue [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    selectedProvider: computed(() => collectorFormState.provider),
+});
+
+const handleChangeProvider = (provider) => {
+    collectorFormStore.setProvider(provider);
+};
+
+
+(() => {
+    collectorFormStore.setProvider(state.providerList[0].name);
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/collector-create/modules/MultipleProviderForm.vue:76]: nitpick: To handle the case where the `provider` cannot be retrieved for any reason...

```typescript
collectorFormStore.setProvider(state.providerList?.[0].name);
```

**Review by yuda110 — APPROVED** [review]
LGTM!

File: apps/web/src/services/asset-inventory/collector/collector-create/modules/MultipleProviderForm.vue [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/collector-create/modules/MultipleProviderForm.vue:76]: suggestion:
```suggestion
    collectorFormStore.setProvider(state.providerList[0]?.name);
```

File: apps/web/src/services/asset-inventory/collector/collector-create/modules/MultipleProviderForm.vue [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+                         class="provider-item"
+                         @change="handleChangeProvider"
+                >
+                    <div class="content-menu-item">
+                        {{ provider.label }}
+                    </div>
+                </p-radio>
+            </p-radio-group>
+        </div>
+        <!-- MEMO: screen mobile size-->
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/collector-create/modules/MultipleProviderForm.vue:21]: suggestion: comment header MEMO -> NOTE

File: apps/web/src/services/asset-inventory/collector/collector-create/modules/MultipleProviderForm.vue [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+import { store } from '@/store';
+
+import type { ProviderReferenceMap } from '@/store/modules/reference/provider/type';
+
+import {
+    useCollectorFormStore,
+} from '@/services/asset-inventory/collector/shared/collector-forms/collector-form-store';
+
+const collectorFormStore = useCollectorFormStore();
+const collectorFormState = collectorFormStore.$state;
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/collector-create/modules/MultipleProviderForm.vue:55]: suggestion: Please move `MultipleProviderForm.vue` to under `shared/collector-form/` for consistency!

File: apps/web/src/services/asset-inventory/collector/collector-create/modules/MultipleProviderForm.vue [hunk 6]
```diff
@@ -0,0 +1,114 @@
+<template>
+    <div class="multiple-provider-form">
+        <p-field-title class="title">
+            Provider
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/collector-create/modules/MultipleProviderForm.vue:4]: suggestion: add comment or apply translation

**Review by WANZARGEN — APPROVED** [review]
Looks good overally. 
I left some suggestions.

File: apps/web/src/services/asset-inventory/collector/collector-create/modules/MultipleProviderForm.vue (hunk 5 above)
Comment (sulmoJ) [reply apps/web/src/services/asset-inventory/collector/collector-create/modules/MultipleProviderForm.vue:55]: This component is not a shared component!

**Pull request merged**